WooCommerce/Classes/ViewRelated/Dashboard/Settings/In-Person Payments/InPersonPaymentsViewController.swift

```diff
@@ -70,6 +70,8 @@ struct InPersonPaymentsView: View {
             switch url {
             case InPersonPaymentsSupportLink.supportURL:
                 showSupport?()
+            case InPersonPaymentsLearnMore.learnMoreURL:
+                if let url = viewModel.learnMoreURL { showURL?(url) }
```

Comment:
```suggestion
                if let url = viewModel.learnMoreURL {
                    showURL?(url)
                }
```

Our iOS coding style [suggests newlines for braces](https://github.com/woocommerce/woocommerce-ios/blob/trunk/docs/coding-style-guide.md#braces), except for `guard let self = self else { return }` as an exception when unwrapping a weak `self` reference.